website/docs/r/virtual_network.html.markdown

```diff
@@ -75,6 +75,8 @@ The following arguments are supported:
 * `guest_mtu` - (Optional) MTU of the network caord on the virtual machine. **Cannot be greater than `mtu`**. Defaults to `1500`. Conflicts with `reservation_vnet` and `reservation_size`.
 * `gateway` - (Optional) IP of the gateway. Conflicts with `reservation_vnet` and `reservation_size`.
 * `network_mask` - (Optional) Network mask. Conflicts with `reservation_vnet` and `reservation_size`.
+* `network_address` - (Optional) Network address. Conflicts with `reservation_vnet` and `reservation_size`.
+* `search_domain` - (Optional) Search Domain. Conflicts with `reservation_vnet` and `reservation_size`.
```

Comment:
could you add some more words ?
[see ONE doc description](https://docs.opennebula.io/6.2/management_and_operations/references/vnet_template.html)